packages/opentelemetry-semantic-conventions/src/resource/SemanticResourceAttributes.ts

```diff
@@ -14,155 +14,837 @@
  * limitations under the License.
  */
 
+import { createConstMap } from '../internal/utils';
+import {
+  DOT,
+  UNDERSCORE,
+  ACCOUNT_DOT,
+  CLOUD_DOT,
+  CONTAINER_DOT,
+  CLUSTER_DOT,
+  DEPLOYMENT_DOT,
+  DEVICE_DOT,
+  ECS_DOT,
+  EKS_DOT,
+  EXECUTABLE_DOT,
+  FAAS_DOT,
+  GROUP_DOT,
+  K8S_DOT,
+  LOG_DOT,
+  IMAGE_DOT,
+  MODEL_DOT,
+  NODE_DOT,
+  OS_DOT,
+  PROCESS_DOT,
+  RUNTIME_DOT,
+  SDK_DOT,
+  SERVICE_DOT,
+  SET_DOT,
+  STREAM_DOT,
+  TASK_DOT,
+  TELEMETRY_DOT,
+  WEBENGINE_DOT,
+  ARN,
+  AWS,
+  AZURE,
+  COMMAND,
+  CRON,
+  DAEMON,
+  DESCRIPTION,
+  HOST,
+  ID,
+  INSTANCE,
+  JOB,
+  NAME,
+  REPLICA,
+  SPACE,
+  STATEFUL,
+  TYPE,
+  UID,
+  VERSION,
+} from '../internal/constants';
+
+//----------------------------------------------------------------------------------------------------------
 // DO NOT EDIT, this is an Auto-generated file from scripts/semconv/templates//templates/SemanticAttributes.ts.j2
-export const SemanticResourceAttributes = {
+//----------------------------------------------------------------------------------------------------------
+
+//----------------------------------------------------------------------------------------------------------
+// Constant values for SemanticResourceAttributes
+//----------------------------------------------------------------------------------------------------------
+
+/**
+ * Name of the cloud provider.
+ */
+export const SEMRESATTRS_CLOUD_PROVIDER = (CLOUD_DOT +
+  'provider') as 'cloud.provider';
```

Comment:
Need to use the `as 'xxxxx'` so that typescript see this constant as that value and not just a constructed `string` (even though all of the values are constant -- it should have figured it out -- but it doesn't